Maifolds/Project.toml

```diff
@@ -0,0 +1,2 @@
+[deps]
```

Comment:
Why do you have the `Maifolds/Project.toml` file? There is a typo.

Reply:
Maybe it was a little late, I'll check and remove it.To clarify: Manifold are – of course – all manifolds that are written in May ;)

Reply:
I'm waiting for some exciting Novemberfolds then ;).